scripts/e2e-modalities.sh

```
@@ -0,0 +1,449 @@
+      append_root "${USERPROFILE}/Downloads/maple_bundle_root"
+    fi
+    ;;
+esac
+
+scan_root() {
+  local root="$1"
+  local dir
+  while IFS= read -r dir; do
+    [[ -n "${dir}" ]] || continue
+    case "${dir}" in
+      *.mlmodelc|*/Unet.mlmodelc)
```

Shellcheck (0.11.0)

[warning] This pattern always overrides a later one on line 253.

(SC2221)